pyccl/tests/test_parameters.py

```diff
@@ -398,18 +392,6 @@ def test_parameters_missing():
         df_mg=None)
 
 
-def test_parameters_set():
-    """
-    Check that a Cosmology object doesn't let parameters be set.
-    """
-    params = ccl.Cosmology(
-        Omega_c=0.25, Omega_b=0.05, h=0.7, A_s=2.1e-9,
-        n_s=0.96)
-
-    # Check that error is raised when unrecognized parameter requested
-    assert_raises(KeyError, lambda: params['wibble'])
-
-
```

Comment:
Why did we lose this?

Reply:
I removed the custom message because Python's default is good! The difference is that it gives you a more relevant `AttributeError` telling you that `'wibble'` doesn't exist.